src/utils.py

```diff
@@ -55,8 +57,15 @@ def random_sleep():
     print("Sleep for " + str(delay) + (delay == 1 and " second" or " seconds"))
     sleep(delay)
 
+def screen_care():
+    d = u2.connect() #connect to device
+    if d.info['screenOn'] == False:
+        print("Turning ON screen")
+        sleep(2)
+        d.press("power")
 
 def open_instagram(device_id):
+    screen_care()
```

Comment:
I think sometimes the instagram is closed and reopened instantly if I'm not mistaken. Maybe @alexal1 can confirm this. But if that is the case, please guarantee that there won't be unnecessary turning the screen off/on.

Reply:
The "screen_care" function get executed everytime the functions "open / close instagram" are called. 
You get the behavior you said only when it crashes. (and it's not instant, there some sleeps between)

To be honest I don't think it's a problem but I can manage that as well.

Reply:
Consider two things:

1. We wouldn't want the screen flickering when instagram crashes
2. What if people are using uiatomator1? Will it crash since it doesn't seem to be backward compatible?Please, check device_facade.py to see a pattern on how to support both uiautomator1 and 2.
3. Maybe this should be optional with default value true. If people don't want this feature for whatever reason they could set it to false.